vendor/ethereum_support/src/network.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Deserialize, Serialize, Hash)]
+#[serde(rename_all = "lowercase")]
+pub enum Network {
+    Mainnet,
+    Regtest,
```

Comment:
❓ Is Regtest have a special designation as a network, like with special network parameters?

Reply:
Ethereum pre-se doesn't have something like a `regtest` mode. I think parity calls it `development`.

Reply:
I was just going with the names included in the registry.